### PR TITLE
New version: PicGo.PicGo version 2.3.1-beta.2

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/40068)

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
`PicGo.PicGo` has stable only releases, so this beta version might need to be moved to `PicGo.PicGo.Beta`.

@Molunerfinn flagged `2.3.1-beta.2` as a release instead of pre-release. This is from the beta release description:
`Note: This is not the official version, you can try new features, but it does not guarantee stability.`

**Comment by Molunerfinn** [comment]
My fault, 2.3.1-beta.2 now is pre-release

File: manifests/p/PicGo/PicGo/Beta/2.3.1-beta.2/PicGo.PicGo.Beta.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+PackageIdentifier: PicGo.PicGo.Beta
+PackageVersion: 2.3.1-beta.2
+PackageLocale: en-US
+Publisher: PicGo
+PublisherUrl: https://github.com/Molunerfinn/PicGo
+PublisherSupportUrl: https://github.com/Molunerfinn/PicGo/issues
+# PrivacyUrl: 
+Author: Molunerfinn
+PackageName: PicGo
```
Comment (ItzLevvie) [inline manifests/p/PicGo/PicGo/Beta/2.3.1-beta.2/PicGo.PicGo.Beta.locale.en-US.yaml:12]:
```suggestion
PackageName: PicGo Beta
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/p/PicGo/PicGo/Beta/2.3.1-beta.2/PicGo.PicGo.Beta.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+PackageUrl: https://molunerfinn.com/PicGo/
+License: MIT License
+LicenseUrl: https://raw.githubusercontent.com/Molunerfinn/PicGo/dev/LICENSE
+Copyright: Copyright (c) 2017-present, Molunerfinn
+CopyrightUrl: https://raw.githubusercontent.com/Molunerfinn/PicGo/dev/LICENSE
+ShortDescription: A simple & beautiful tool for pictures uploading built by vue-cli-electron-builder
+Description: A simple & beautiful tool for pictures uploading built by vue-cli-electron-builder
+Moniker: picgo
```
Comment (ItzLevvie) [inline manifests/p/PicGo/PicGo/Beta/2.3.1-beta.2/PicGo.PicGo.Beta.locale.en-US.yaml:20]:
```suggestion
Moniker: picgo-beta
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/p/PicGo/PicGo/Beta/2.3.1-beta.2/PicGo.PicGo.Beta.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+InstallerType: nullsoft
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://github.com/Molunerfinn/PicGo/releases/download/v2.3.1-beta.2/PicGo-Setup-2.3.1-beta.2-ia32.exe
+  InstallerSha256: 195B599CEB5C3221FC7AC855B542A4D1D4BD882660BB602A94654D03B5A0D1B0
```
Comment (ItzLevvie) [inline manifests/p/PicGo/PicGo/Beta/2.3.1-beta.2/PicGo.PicGo.Beta.installer.yaml:17]:
```suggestion
  InstallerUrl: https://github.com/Molunerfinn/PicGo/releases/download/v2.3.1-beta.2/PicGo-Setup-2.3.1-beta.2-x64.exe
  InstallerSha256: 2B3010C65FAC4337883A04E2099729E92024B28380D5B6B892E131046690535C
```

File: manifests/p/PicGo/PicGo/Beta/2.3.1-beta.2/PicGo.PicGo.Beta.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: x64
+  Scope: machine
+  InstallerUrl: https://github.com/Molunerfinn/PicGo/releases/download/v2.3.1-beta.2/PicGo-Setup-2.3.1-beta.2-ia32.exe
+  InstallerSha256: 195B599CEB5C3221FC7AC855B542A4D1D4BD882660BB602A94654D03B5A0D1B0
```
Comment (ItzLevvie) [inline manifests/p/PicGo/PicGo/Beta/2.3.1-beta.2/PicGo.PicGo.Beta.installer.yaml:23]:
```suggestion
  InstallerUrl: https://github.com/Molunerfinn/PicGo/releases/download/v2.3.1-beta.2/PicGo-Setup-2.3.1-beta.2-x64.exe
  InstallerSha256: 2B3010C65FAC4337883A04E2099729E92024B28380D5B6B892E131046690535C
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/p/PicGo/PicGo/Beta/2.3.1-beta.2/PicGo.PicGo.Beta.installer.yaml (hunk 4 above)
Comment (Molunerfinn) [reply manifests/p/PicGo/PicGo/Beta/2.3.1-beta.2/PicGo.PicGo.Beta.installer.yaml:23]: using x64 version is OK

**Review by Molunerfinn — APPROVED** [review]
done

**Review by Molunerfinn — APPROVED** [review]

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**